### ISSUE TITLE
Error installing gridle

### ISSUE THREAD

**Issue by yeltsinseal** [Coffeekraken/gridle#126]
Hello,

I'm using Windows 10 x64 with node v7.10.0 and npm v.4.2.0. All extensions I tried worked fine, but then I tried to install gridle and it shows me this:

```
C:\Bitnami\wampstack-7.1.4-0\apache2\htdocs\yeltsinseal\wp-content\themes\yeltsinseal>npm install coffeekraken-gridle --save

> coffeekraken-gridle@3.0.0-rc.1 install C:\Bitnami\wampstack-7.1.4-0\apache2\htdocs\yeltsinseal\wp-content\themes\yeltsinseal\node_modules\coffeekraken-gridle
> ./scripts/install.sh

'.' não é reconhecido como um comando interno
ou externo, um programa operável ou um arquivo em lotes.
npm ERR! Windows_NT 10.0.15063
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install" "coffeekraken-gridle" "--save"
npm ERR! node v7.10.0
npm ERR! npm  v4.2.0
npm ERR! code ELIFECYCLE
npm ERR! errno 1

npm ERR! coffeekraken-gridle@3.0.0-rc.1 install: `./scripts/install.sh`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the coffeekraken-gridle@3.0.0-rc.1 install script './scripts/install.sh'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the coffeekraken-gridle package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     ./scripts/install.sh
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs coffeekraken-gridle
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls coffeekraken-gridle
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     C:\Users\yelts\AppData\Roaming\npm-cache\_logs\2017-06-11T21_33_44_623Z-debug.log
```

The logs can be found here:
[_logs.zip](https://github.com/Coffeekraken/gridle/files/1066777/_logs.zip)

Any idea what can it be?

**Comment by kayuda91** [comment]
I have same error on Windows 10 x64. Node v6.10.3 and npm v3.10.10.

**Comment by tymothytym** [comment]
For me this was resolved by reinstalling git (2.14.2). Windows 10 messing with the PATH variables after an automatic update meant that `sh` was no longer a recognised command.

**Comment by olivierbossel** [comment]
Hi guys,
Thanks for your feedback. This is not an issue relative to gridle but it's related to node. Try to reinstall your node version or upgrade it and give it another chance.
Thanks for your feedback again!
